tests/acceptance/init-test.js

```diff
@@ -166,4 +166,67 @@ describe('Acceptance: ember init', function () {
 
     expect(dir('.git')).to.not.exist;
   });
+
+  // [WIP] ember init --lang flag
+  // -------------------------------
+  // Good: Default
+  it('ember init without --lang flag (default) has no lang attribute in index.html', async function () {
+    await ember(['init', '--skip-npm', '--skip-bower', '--skip-git']);
+    confirmBlueprinted();
+    // expect(file('app/index.html')).to.contain('<html>');
+  });
+
+  // Good: Correct Usage
+  it('ember init with --lang flag and valid code assigns lang attribute in index.html', async function () {
+    await ember(['init', '--skip-npm', '--skip-bower', '--skip-git', '--lang=en-US']);
+    // expect(file('app/index.html')).to.contain('<html lang="en-US">');
```

Comment:
Should probably be uncommented 😄

Reply:
😄 Going to be hard to beat that one! Removed as part of 72e10e9 👍